components/refiner/actions/identify-user/identify-user.mjs

```diff
@@ -0,0 +1,43 @@
+import { ConfigurationError } from "@pipedream/platform";
+import refiner from "../../refiner.app.mjs";
+
+export default {
+  key: "refiner-identify-user",
+  name: "Identify User",
+  description: "Creates or updates a user profile in Refiner. [See the documentation](https://refiner.io/docs/api/#identify-user)",
```

Comment:
This description doesn't make much sense - 'identify' seems like a GET, not a create or update